cmd/oras/root/manifest/fetch.go

```diff
@@ -93,7 +93,7 @@ Example - Fetch raw manifest from an OCI layout archive file 'layout.tar':
 			opts.RawReference = args[0]
 			return option.Parse(cmd, &opts)
 		},
-		Aliases: []string{"get"},
+		Aliases: []string{"get", "show"},
```

Comment:
Should we add a simple e2e test for the new alias?

Reply:
added